packages/common-resources/static-resources/src/i18n/en_US/common.ts

```diff
@@ -177,7 +178,7 @@ export default {
     labelConnectBy: 'Connected with <1>{{connectBy}}</1>',
     labelWrongNetwork: 'Wrong network',
     labelActivatedAccountDeposit: 'Please deposit to activate your Layer2 account',
-    labelActivatedAccountNotSupport: 'We are apologize, Loopring app not support your address current now.',
+    labelActivatedAccountNotSupport: 'We apologize, loopring app does not support your address currently.',
```

Comment:
```suggestion
    labelActivatedAccountNotSupport: 'We apologize, loopring app does currently not support your address.',
```

Reply:
We apologize, loopring app currently does not support your address.